RayLin.md

```diff
@@ -1,16 +1,18 @@
-# 05/23 ~ 05/27
+# 05/30 ~ 06/03
 
 - [Firefox for Android]
   - Bug 1086911 - Update style of about:firefox
-    - ask visual for feedback
-  - Bug 1091241 - Make add-on detail page look more like a detail page
-    - FIXED
+    - feedback-
+    - issues: blurry logo and distorted icon
   - Bug 1267935 - Video controls are not correctly scaled for several video types
-    - uplifted to Aurora
-    - video control is scaled fine in latest Aurora build(05/29) now
+    - uplift to mozilla-beta? next week
+  - Bug 1276686 - Selection handles are displayed when tapping on the reader view toolbar
+    - checkin-needed
 - [Firefox]
   - Bug 887934 - [webvtt] Update \<video\> controls to include options for closed captioning
-    - r? UI issues are fixed in the main
-    - next week:
-      - ask visual for new CC button icon, since transparency PNG could not be brighter even opacity is set to 1
-      - rewrite Part 2 to align with WebVTT spec
+    - r+
+    - will update patch and ask for checkin once got new icon from :shorlander
```

Comment:
Sure! Though :shorlander is on https://bugzilla.mozilla.org/show_bug.cgi?id=1271765, I don't think it could be done in the near future.